ui/src/components/toast-error.tsx

```diff
@@ -20,7 +20,7 @@
 import { ApiError } from '@/api/requests';
 
 type ToastErrorProps = {
-  error: ApiError;
+  error: unknown;
```

Comment:
It seems like in v5, TanStack Query has changed the default error type [from unknown to Error](https://tanstack.com/query/latest/docs/framework/react/guides/migrating-to-v5#typescript-error-is-now-the-default-type-for-errors-instead-of-unknown), so just a heads up, this might change once the codegen library is updated.

Reply:
I don't think that that will be a problem, I think this will accept an error of type Error too.